src/js/hora.js

```diff
@@ -335,196 +383,11 @@ define([
             }
         };
 
-        Hora.navigation = {
```

Comment:
https://github.com/mobify/hora.js/wiki

You can edit it directly, or clone it (wiki's are git repos)